components/card.css

```diff
@@ -0,0 +1,231 @@
+.card{    
+    width: 35rem;
+    border-radius: 4px;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+    outline: 1px solid var(--secondary-colour);
+}
+
+.card-box-shadow{
+    outline: none;
+    box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%), 0 1px 1px 0 rgb(0 0 0 / 14%), 0 1px 3px 0 rgb(0 0 0 / 12%);
+}
+
+.card-img{
+    object-fit: cover;
+    width: 35rem;
+    height: 20rem;
+}
+
+.card-img-border{    
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+}
+
+.card-title-box{
+    padding: 1.6rem;
+}
+
+.card-title{
+    font-size: 2rem;
+    line-height: 3.2rem;
+}
+
+.card-subtitle{
+    color: rgba(0,0,0,.54);
+    font-size: 1.4rem;
+    line-height: 2.2rem;
+    letter-spacing: .0071428571px;
```

Comment:
you can increase the value of letter-spacing lil bit more it will look nice

Reply:
one place after decimal works well

Reply:
Sure.